### ISSUE TITLE
[Accessibility][Usable - Property Pane_Ask a question (Bot Asks)]: Screen reader announces the placeholder multiple times.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2122]
**User Experience** 
Users who rely on AT will might get confused if the same information is announced multiple times.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE and turn on narrator.
2. Navigate to already created dialog tree item and activate it. Such as- Greetings, Actions, EndDialog etc.
3. Click on "+" and select 'Ask a question'. Then select any of the available options except Oauth login.
4. Navigate to property pane appeared at right side of the screen.
5. Now, navigate to 'Default value' edit field and observe narrator narration.
6. Observe the issue.

**Actual:** 
The narrator narrates the placeholder twice.
Ex- "ex. @username, coalesce (@number, @partysize)" 

**Expected:**
The narrator should announce the placeholder only once.

**Note:**
1. Same issue is observed for all the Property Pane which have placeholder at edit fields appeared at right side of the screen.
2. The narrator still narrates placeholder even the edit field have text content in it.

**Observation:**
Same issue observed with NVDA.

**MAG Reference:** [MAG](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7BF98F4235-B6BD-47D3-8DA4-F28B9B6FBFA8%7D&file=Guideline%20101%20%E2%80%93%20Recommended%20Assistive%20Technology%20Tools%20for%20MAS%20Testing%20of%20Microsoft%20Digital%20Properties.docx&action=default&mobileredirect=true&cid=cfd410f9-ab7d-45d6-b822-faae4139f746)

**Attachment:**
[Usable_narrator announces the placeholder multiple times.zip](https://github.com/microsoft/BotFramework-Composer/files/4265261/Usable_narrator.announces.the.placeholder.multiple.times.zip)
![placeholder are narrated](https://user-images.githubusercontent.com/60913131/75660281-85060900-5c91-11ea-9060-cb3568c3c0dc.png)

**Comment by ashish315** [comment]
@beyackle  As checked and verified on below environment, the issue is still repro. I downloaded below Master build file last updated on 18th April.

**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)
![image](https://user-images.githubusercontent.com/60913131/79755879-6c33de80-8337-11ea-90cd-8dd51ac56600.png)

**Comment by beyackle** [comment]
I'm picking this back up again to see if we can finally close it.

**Comment by ashish315** [comment]
As checked still the issue is not resolved. 

**Test Environment**
Browser: Microsoft Edge Canary Version: 85.0.531.1 (Official build)
OS build: 2004 (19635.1)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)
Master folder downloaded on 05-06-2020

**Comment by beyackle** [comment]
Okay. Taking another look at this so we can finally resolve it (although it's marked A11yUsable, so this might end up as a "won't fix" in the end).

**Comment by beyackle** [comment]
@ashish315 This is a problem with Fluent's TextField component, not with Composer. See the examples at https://developer.microsoft.com/en-us/fluentui#/controls/web/textfield - the "With placeholder" field also gets its placeholder ("Please enter text here") read twice. We import TextField as-is.

**Comment by beyackle** [comment]
I've opened an issue in Fluent's repo about this. https://github.com/microsoft/fluentui/issues/13515

**Comment by cwhitten** [comment]
@ashish315 can you close this on our audit given that it is due to a dependency?

**Comment by ashish315** [comment]
@cwhitten Agree with you. Closing the bug for now will track the bug according to Fluent's  UI.
